lib/setter.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+exports.boolean = (value) => {
+  switch (value) {
+    case 'true': case 'on': case true:
+      return true
+    case 'false': case 'off': case null: case false:
+      return false
+    default:
+      throw new TypeError('Invalid value. Valid options are: a boolean value')
+  }
+}
+
+exports.enum = (value) => {
+  return value
+}
+
+exports.numeric = (value) => {
+  let n = Number(value)
+  if (isNaN(n)) {
```

Comment:
Maybe `if (!isFinite(n)) {` instead? I doubt we have much use for an `Infinity` or `-Infinity` value for settings, which this still allows.